website/vercel.json

```diff
@@ -2,6 +2,11 @@
   "cleanUrls": true,
   "trailingSlash": false,
   "redirects": [
+    {
+      "source": "/src/pages/styles.js",
+      "destination": "https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/adding-page-components.md",
+      "permanent": true
```

Comment:
It looks like the redirect isn't being picked up here: https://docs-getdbt-com-git-remove-style-dbt-labs.vercel.app/styles

The `source` here just needs to match the live url of the page, which would be `/styles` in this case!

Reply:
thanks so much for reviewing @JKarlavige ! I've updated this on my end and ready for re review whenever you can 🙏